scripts/netContain/scripts/ovsInit.sh

```diff
@@ -27,6 +28,8 @@ ovsdb-server --remote=punix:/var/run/openvswitch/db.sock --remote=db:Open_vSwitc
 echo "Starting ovs-vswitchd"
 ovs-vswitchd -v --pidfile --detach --log-file=/var/contiv/log/ovs-vswitchd.log -vconsole:err -vsyslog:info -vfile:info &
 
+sleep 2
```

Comment:
wait for port open?

Reply:
Is there a simple `ovs-vsctl` command we can run here to poll instead of sleeping?

Reply:
done